easybuild/easyconfigs/s/stardist/stardist-0.8.3-foss-2021b-CUDA-11.4.1.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'stardist'
+version = '0.8.3'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/stardist/stardist'
+description = "Object Detection with Star-convex Shapes."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('TensorFlow', '2.7.1', versionsuffix),
+    ('numba', '0.54.1'),
+    ('imageio', '2.13.5'),
+    ('scikit-image', '0.19.1'),
+    ('tqdm', '4.62.3'),
+    ('PyOpenCL', '2021.2.13', versionsuffix),
+    ('Mako', '1.1.4'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tifffile', '2022.10.10', {
+        'checksums': ['50b61ba943b866d191295bc38a00191c9fdab23ece063544c7f1a264e3f6aa8e'],
+    }),
+    ('csbdeep', '0.7.2', {
+        'checksums': ['8a000eb71d04aa753f52ffe81a34c0a30a06ee986d20b9986d76272253e2fd53'],
+    }),
+    ('edt', '2.3.0', {
+        'checksums': ['220e2086fdf32bbd8964df76b3e1a42061ece1f4b9ea95ec83a094d24d258664'],
+    }),
+    ('configparser', '5.3.0', {
+        'checksums': ['8be267824b541c09b08db124917f48ab525a6c3e837011f3130781a224c57090'],
+    }),
+    ('funcsigs', '1.0.2', {
+        'checksums': ['a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50'],
+    }),
+    ('reikna', '0.7.6', {
+        'checksums': ['722fefbd253d0bbcbf5250b7b9c4aca5722cde4ca38bfbf863a551a5fc26edfa'],
+    }),
+    ('scikit-tensor-py3', '20210331', {
+        'modulename': 'sktensor',
+        'sources': [{'filename': 'scikit-tensor-py3.tar.gz',
+                     'git_config': {'url': 'https://github.com/evertrol',
+                                    'repo_name': 'scikit-tensor-py3',
+                                    'commit': '99190292b9564e7c4a9312280db732ceedf7b7e3',
+                                    },
+                     }],
+        'checksums': ['a2765cc930e5937c8358fe79daf37bb2c0d1470f0b1e4daf6c4082c12195e472'],
```

When using `git_config`, we can't add a checksum since the tarball is created on the fly in a non-reproducible way currently (until gets merged), so we should use `None` as checksum.

But in fact, there's no need to use `git_config` at all here, since a source tarball can be obtained by GitHub directly for a specific commit, via https://github.com/evertrol/scikit-tensor-py3/archive/99190292b9564e7c4a9312280db732ceedf7b7e3.tar.gz in this case.

Via `download_filename` and `filename` we can then get a proper filename for the tarball as well.

```suggestion
        'sources': [{
            'source_urls': ['https://github.com/evertrol/scikit-tensor-py3/archive/'],
            'download_filename': '99190292b9564e7c4a9312280db732ceedf7b7e3.tar.gz',
            'filename': 'scikit-tensor-py3-%(version)s.tar.gz',
        }],
        'checksums': ['5afb33e74c7e7401374a40db4b381b3da182cc76a9b6282bdf04cafff097c046'],
        'modulename': 'sktensor',
```